Actions/AL-Go-Helper.ps1

```diff
@@ -1952,3 +1952,44 @@ function GetProject {
     }
     $project
 }
+
+function Get-NavSipFromArtifacts() {
```

Comment:
Generally, let's try to avoid adding functions in here as such helper files tend to grow very quickly.

In this case, do you think that this functionality (`Get-NavSipFromArtifacts` or `Register-NavSip`) would be reused in other places, other than signing? If not, fear not to simply move these functions in scripts, specific for the action itself.